### ISSUE TITLE
DocBuilder needs to pass --config baseurl=...

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3130]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3130|
|Type|defect|
|Created|2008-03-27 18:38:56Z|
|Branch|https://github.com/twisted/twisted/tree/docbuilder-baseurl-3130|

API links are incorrect because we're not specifying the URL format string for generating them.

<details><summary>Searchable metadata</summary>

```
trac-id__3130 3130
type__defect defect
reporter__radix radix
priority__normal normal
milestone__ 
branch__branches_docbuilder_baseurl_3130 branches/docbuilder-baseurl-3130
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1206643136000000 1206643136000000
changetime__1206968532000000 1206968532000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [23111]) Branching to 'docbuilder-baseurl-3130'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

Ready for review

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @exarkun|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [23149]) Merge docbuilder-baseurl-3130

Author: exarkun, itamarst
Reviewer: therve
Fixes [#3130](https://github.com/twisted/twisted/issues/3130)

Add support to the release automation classes for specifying where API links in the
lore-processed documentation point in the output.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
